src/util/field.rs

```diff
@@ -3,26 +3,42 @@
 // NOTE: when we eventually break CirC into separate crates,
 //       each crate may want its own field type default
 
```

Comment:
@kwantam Do you have time to look at the changes to this file? I think you have the best perspective.